language/move-ir-compiler/move-ir-to-bytecode/src/parser.rs

```diff
@@ -105,4 +105,14 @@ mod tests {
             good_chars.pop();
         }
     }
+
+    #[test]
```

Comment:
Gah! I'm super sorry. I was reminded that we actually have these `.exp` tests for the IR now too. We should make this one to follow suit. 

It should go in 
`language/move-ir-compiler/transactional-tests/tests/parsing`